x-pack/plugins/enterprise_search/public/applications/workplace_search/types.ts

```diff
@@ -127,7 +129,44 @@ interface SourceActivity {
   status: string;
 }
 
-interface IndexingConfig {
+export interface SyncEstimate {
+  duration?: string;
+  nextStart: string;
+  lastRun?: string;
+}
+
+interface SyncIndexItem<T> {
+  full: T;
+  incremental: T;
+  delete: T;
+  permissions?: T;
+}
+
+interface IndexingSchedule extends SyncIndexItem<string> {
+  estimates: SyncIndexItem<SyncEstimate>;
+}
+
+export type SyncJobType = 'full' | 'incremental' | 'delete' | 'permissions';
+
+export const DAYS_OF_WEEK_VALUES = [
+  'sunday',
+  'monday',
+  'tuesday',
+  'wednesday',
+  'thursday',
+  'friday',
+  'saturday',
+] as const;
```

Comment:
@bhuvanaurora in the Day of week multiselect, what day should be the first one in the list? Sunday or Monday?
I'm talking about this multiselect:
![image](https://user-images.githubusercontent.com/11838280/135940334-1d045fc4-ae21-4565-9a66-f41412c96b3e.png)

Reply:
This is possibly going away, as the API does not support it.